### PR TITLE
Added version labels, fixed PATH, jboss user chown

### PR DESCRIPTION
**Pull request by dhirajsb**

### CODE REVIEW

File: sti/karaf/Dockerfile [hunk 1]
```diff
@@ -38,10 +41,11 @@ COPY ./jolokia.properties /opt/jolokia/jolokia.properties
 COPY ./deploy-and-run.sh /opt/karaf/bin/
 RUN chmod a+x /opt/karaf/bin/deploy-and-run.sh
 
-# allow jboss user access to /opt/jboss
-RUN chown -R 1000 /opt/jboss
+# allow jboss user access to /opt/jboss and /opt/karaf
+RUN chown -R 1000 /opt/jboss \
```
Comment (jimmidyson) [inline sti/karaf/Dockerfile]: If these dirs need to be writeable by the user at runtime then you need to `chmod 777` them & `chmod 666` any non-executable files. The jboss user will not be used at runtime, rather an arbitrary uid will be used.

Comment (dhirajsb) [inline sti/karaf/Dockerfile]: Yeah, that's being replaced with making the dirs world writable according to openshift guidelines

**Comment by jimmidyson** [comment]
Do we need to be so loose to allow 777 on `/opt/jboss`? Should we be more restrictive to only those dirs that should actually be writeable?

**Comment by dhirajsb** [comment]
I know, apparently that's what EAP images do too, which is absolutely un-secure. But the directory permissions are set before the karaf archive has been extracted. So, hopefully after extraction file permissions from the archive are not so loose.

**Pull request merged**